tasks/git-advanced.md

```diff
@@ -0,0 +1,140 @@
+# Advanced git
+**Enhance your git skills**
+
+> This task is optional.
+> You don't need it to advance further in the course.
+> However, you will definitely apply newly acquired skills
+> in your future work.
+
+## Join our chat: [![Gitter](https://badges.gitter.im/Kottans/frontend.svg)](https://gitter.im/Kottans/frontend?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
+
+[![MIT Licensed](https://img.shields.io/badge/license-MIT-blue.svg)](https://github.com/Kottans/web/blob/master/LICENSE.md)
+[![Ideas and useful links](https://img.shields.io/badge/google--doc-ideas-ff69b4.svg)](https://docs.google.com/spreadsheets/d/1bZJhYjK3VHOS2HmQb2Fs4aHfEBt8mp1F09j9nEEDaqE/edit#gid=818017811)
+[![Awesome](https://cdn.rawgit.com/sindresorhus/awesome/d7305f38d29fed78fa85652e3a63e154dd8e8829/media/badge.svg)](https://github.com/sindresorhus/awesome#front-end-development)
+
+## Pre-requisites
+
+You already some have hands-on experience with:
+ * [ ] creating, forking and cloning repos
```

Comment:
Any particular reason to use checkboxes here? I feel like bullet points would be a better option semantically. 
Just a suggestion, otherwise lgtm

Reply:
Checkpoints are good but I think it's better to use only single thing: checkpoints or bullets.

Reply:
It's a github flavoured markdown syntax. It renders like following:
 * [ ] item 1
 * [ ] item 2

BTW, `* [x]...` renders as
 * [x] item X

Idea behind checkboxes is to give a reader a hint to do a self check against the list.

I am pretty OK with pure bullets either. @viktor-yakubiv what do you think?

Reply:
I see. Looks fine. I'd rather leave as is, with checkboxes.